textproc/lilypond/Portfile

```diff
@@ -84,11 +85,11 @@ depends_build-append \
                     port:flex \
                     port:fontforge \
                     port:gmake \
-                    port:p5.28-encode \
-                    port:p5.28-pod-escapes \
-                    port:p5.28-pod-simple \
-                    port:p5.28-podlators \
-                    port:p5.28-scalar-list-utils \
+                    port:p${perl5.major}-encode \
+                    port:p${perl5.major}-pod-escapes \
+                    port:p${perl5.major}-pod-simple \
+                    port:p${perl5.major}-podlators \
+                    port:p${perl5.major}-scalar-list-utils \
```

Comment:
Doesn't seem to be telling the build which perl to use.

Reply:
Noted. Will dig in.